app/assets/javascripts/discourse/app/templates/components/time-shortcut-picker.hbs

```diff
@@ -20,8 +20,10 @@
           </div>
         </div>
         <div class="control-group custom-date-time-wrap custom-relative-wrap">
-          <label class="control-label">{{i18n "relative_time_picker.relative"}}</label>
-          <RelativeTimePicker @onChange={{action "relativeTimeChanged"}} />
+          <label class="control-label" for="bookmark-relative-time-picker">
+            {{i18n "relative_time_picker.relative"}}
+          </label>
+          <RelativeTimePicker @id="bookmark-relative-time-picker" @onChange={{action "relativeTimeChanged"}} />
```

Comment:
Does that `@id=""` work? Would `id=""` work? My concern is that `RelativeTimePicker` is a classic but tag-less component so I'm not sure either syntax actually applies that id to any element.

Reply:
Yes, it works, and we were already using it in a few other places like: 

https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/templates/components/edit-category-settings.hbs#L122

Reply:
Oh, also, the id is not used in the component itself, but it is passed to the input (which is what we want for accessibility here): https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/templates/components/relative-time-picker.hbs#L2